docs/docfx.json

```diff
@@ -23,7 +23,10 @@
           "src": "../src"
         }
       ],
-      "dest": "api"
+      "dest": "api",
+	  "properties": {
+        "TargetFramework": "netstandard1.6"
```

Comment:
We have. For example right now we dont have Akka.IO on netcore. We don't have some members related to `System.Configuration`